packages/core/src/helper/featureDetection.ts

```diff
@@ -0,0 +1,6 @@
+/**
+ * Returns `true` if the browser supports screen sharing.
+ */
+export function supportsScreenSharing(): boolean {
+  return navigator?.mediaDevices && !!navigator.mediaDevices.getDisplayMedia;
```

Comment:
haven't tested `navigator?.mediadevices` on non browser envs, if you have, feel free to ignore the suggestion
```suggestion
  return typeof navigator !== 'undefined' && navigator.mediaDevices && !!navigator.mediaDevices.getDisplayMedia;
```

Reply:
Good call, better safe than sorry!